cdc/processor/sourcemanager/manager.go

```diff
@@ -85,13 +92,31 @@ func NewForTest(
 	}
 }
 
+func isOldUpdateKVEntry(raw *model.RawKVEntry, thresholdTs model.Ts) bool {
+	return raw != nil && raw.OpType == model.OpTypePut && raw.OldValue != nil && raw.Value != nil && raw.CRTs < thresholdTs
```

Comment:
Maybe we can extract the `RawKVEntry.IsUpdate` method to simplify this.

Reply:
Fixed.